src/auto-evo/RunResults.cs

```diff
@@ -311,6 +323,16 @@ void OutputPopulationForPatch(Species species, Patch patch, long population)
 
                 if (playerReadable)
                     builder.Append("\n");
+
+                if (GetGlobalPopulation(entry.Species) == 0 && !moving)
+                {
+                    // Remove a new line
+                    builder.Remove(builder.Length - 1, 1);
```

Comment:
Instead of removing a new line put this before this block:
```
                if (playerReadable)
                    builder.Append("\n");
```

Reply:
I originally had it like that, but it started acting real funky when I did. It you really want it there, I can put it there though.

Reply:
Can you describe more?
It's pretty weird to me that you would need to remove an already built part of the string, instead of reordering when different parts of the string are built.

Reply:
That problem no longer seems an issue. Strange.